.github/actions/e2e-atomic/action.yml

```diff
@@ -25,7 +25,7 @@ runs:
         browser: chrome
         command-prefix: 'npx -w @coveo/atomic cypress-repeat run -n 3 --until-passes --rerun-failed-only --'
         spec: ${{ inputs.spec }}
-        start: npx nx run atomic:prod
+        start: npm start -w @coveo/atomic
```

Comment:
Every test is failing using atomic:prod (it uses the /www folder). It is probably a bug with stencil and exports field dependencies that they haven't really looked at yet since they have problems with "exports" field dependencies. The /www folder problems will not affect us so we will change our test to use the dev environment (like we do with playwright).